tests/utils/dockercli/volumelifecycle.go

```diff
@@ -171,6 +183,17 @@ func StopContainer(ip, containerName string) (string, error) {
 	return ssh.InvokeCommand(ip, dockercli.StopContainer+containerName)
 }
 
+// AttachVolume - attach volume to container on given host
+func ExecContainer(ip, volName, containerName string) (string, error) {
+	log.Printf("Attaching volume [%s] on VM [%s]\n", volName, ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.RunContainer+" -d --rm -v "+volName+
+			":/vol1 --name "+containerName+ dockercli.TestContainer)
+	if err != nil {
+		return out, err
+	}
+	return ssh.InvokeCommand(ip, dockercli.RemoveContainer + containerName)
```

Comment:
not sure L#194 and L#189 is consistent .. can you please help me to understand why do we need L194?

I would prefer test to invoke `mount volume with --rm flag`  > `detach or exit container which removes automatically`.

I would wait for your response though.